ConsumerApi.Sdk/ConsumerApi.Sdk.csproj

```diff
@@ -5,8 +5,13 @@
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
I would prefer System.Text.Json over Newtonsoft.Json. Is there a reason why you chose Newtonsoft?

Reply:
We're using `JsonConvert.DeserializeObject` in other places, and that's available through `Newtonsoft.Json`. I'm not sure if `System.Text.Json` provides an equivalent API.

Reply:
Okay, then let's keep it for now.